db/models/aggregate_functions.py

```diff
@@ -186,6 +183,16 @@ def update_application_and_related_form(
     application = ApplicationsMethods.get_application_by_id(application_id)
     application.last_edited = func.now()
     form_sql_row = FormsMethods.get_form(application_id, form_name)
+
+    if form_name == "project-information":
+        fields_array = question_json[1]["fields"]
+        for key in fields_array:
+            if key["key"] == "KAgrBz":
```

Comment:
As I am making changes to the forms, the Key might change. What are your thoughts on 
if key["key"] == "KAgrBz" and if key[ "title"] =="Project name":

Reply:
thats a good idea! I like the sound of (if key["key"] == "KAgrBz" or if key[ "title"] =="Project name":) 👍🏾